librz/core/cfile.c

```diff
@@ -101,7 +101,7 @@ RZ_API int rz_core_file_reopen(RzCore *core, const char *args, int perm, int loa
 		// XXX - select the right backend
 		if (core->file) {
 			newpid = rz_io_fd_get_pid(core->io, core->file->fd);
-			newtid = rz_io_fd_get_tid(core->io, core->file->fd);
```

Comment:
this is dead only on linux i think, on other platforms `newtid` is overwritten with `newpid`.

Reply:
PS: rz_io_fd_get_tid has no side effect